ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/SelectOneOptionAdapter.kt

```diff
@@ -29,7 +29,7 @@ import com.google.android.ground.ui.datacollection.tasks.multiplechoice.SelectOn
  */
 class SelectOneOptionAdapter(
   private val options: List<Option>,
-  private val viewModel: MultipleChoiceTaskViewModel
+  private val handleOptionSelected: (List<Option>) -> Unit
```

Comment:
Since this only operates on one value, change the signature to `Option -> Unit`. We can still use the same function in the viewmodel, but I'd change the signature here for the sake of correctness, and this way the adapter does not need to know about the VM impl detail; e.g. in the VM, we can do the list wrapping:

```kotlin
updateResponse(opts: List<Option> ...)

updateResponse(option: Option ...) {
  updateResponse(listOf(option))
}
```